app/views/ProjectEdit/Framework/index.tsx

```diff
@@ -214,8 +218,12 @@ function ProjectFramework(props: Props) {
         offset,
     ]);
 
-    const totalFrameworksCount = frameworksResponse?.count ?? 0;
-
+    const totalFrameworksCount = projectAflist?.analysisFrameworks?.totalCount ?? 0;
+    const afList: FrameworkMini[] = (projectAflist?.analysisFrameworks?.results || [])
+        .map((item) => ({
+            ...item,
+            id: Number(item.id),
+        }));
```

Comment:
Lets not cast id to number.
After adopting graphql, all the id fields are string as well.
As this was in REST before, we used number to save selectedFramework, but it was eventually cast to string while checking whether its active or not.
Lets fix that there, and not cast.

Basically I recommend not casting it and fixing other places it causes issue to.

Reply:
@AdityaKhatri ,
By  "fixing other places it causes issue to" does this mean fix the type issues in ListView in **deep-ui** ?
.